cli/metrics/metrics.go

```diff
@@ -89,3 +97,18 @@ func GetCommand(args []string) string {
 	}
 	return result
 }
+
+type Metadata struct {
+	Build BuildMetadata `json:"build,omitempty"`
```

Comment:
Should it just be a more generic `map[string]interface{}` here to avoid knowing about the internals of the build metrics?

Reply:
It's internal yes but we could reuse this object in an underlying logic that will import this package, let's say on pinata? But I think I should move that to another package under metrics actually like `metadata`.

Reply:
Ok metadata is now under a dedicated package so we can reuse these types on pinata if we want.

Reply:
I don’t really think we’ll do that though, we’re probably more going to just read it as a `map[string]interface{}` and add that to the heartbeat, aren’t we?

Reply:
Oh ok if that's the case it makes more sense to do it this way yes.